ompi/mca/op/rocm/op_rocm.h

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2019-2023 The University of Tennessee and The University
+ *                         of Tennessee Research Foundation.  All rights
+ *                         reserved.
+ * $COPYRIGHT$
+ *
+ * Additional copyrights may follow
+ *
+ * $HEADER$
+ */
+
+#ifndef MCA_OP_CUDA_EXPORT_H
+#define MCA_OP_CUDA_EXPORT_H
+
+#include "ompi_config.h"
+
+#include "ompi/mca/mca.h"
+#include "opal/class/opal_object.h"
+
+#include "ompi/mca/op/op.h"
+#include "ompi/runtime/mpiruntime.h"
+
+#include <hip/hip_runtime.h>
+#include <hip/hip_runtime.h>
```

Comment:
probably don't need to include hip_runtime.h twice,
Also, I would recommend to add the GCC diagnostic disable messages around this include as is done in the accelerator/rocm component, to avoid getting flooded with warnings from the hip header file, it just makes it harder to spot actual issues in the code. See

https://github.com/open-mpi/ompi/blob/main/opal/mca/accelerator/rocm/accelerator_rocm.h#L18

Reply:
That was an artefact of `hipify`. Should be fixed in latest push.